includes/Checker/Checks.php

```diff
@@ -54,7 +54,7 @@ public function run_checks( array $checks ) {
 		$checks_to_run = array_filter(
 			$checks,
 			function( $check ) use ( $all_checks ) {
-				return in_array( $check, $all_checks, true );
+				return in_array( $check, $all_checks ); // phpcs:ignore WordPress.PHP.StrictInArray.MissingTrueStrict
```

Comment:
@felixarntz This change was required in order for the `Check` instances passed to match with those returned in `get_checks()`. I know we have discussed setting up the checks array with slugs for keys to better match with what would be passed via CLI/AJAX.

I can update this if you agree with that approach?

Reply:
We should still change that, but for now, I'd suggest a simpler fix: The only reason the `true` here didn't work is because every `get_checks()` call creates new instances right now. That's somewhat problematic in itself, so what we should probably do is modify `Checks::get_checks()` to create that list (and run the filter) only once. We can just "memoize" the result after the first run in a static or class variable.

Reply:
Thanks @felixarntz I've updated this so we have a single instance of the `Checks` class.